recipes/hlama/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: hlama
-  version: "0.2"
+  version: '0.3'
```

Comment:
Can you please use a stable URL like: https://github.com/bihealth/hlama/archive/v0.3.tar.gz